airbyte-cdk/bulk/core/extract/src/main/kotlin/io/airbyte/cdk/output/CatalogValidationFailureHandler.kt

```diff
@@ -19,59 +20,49 @@ interface CatalogValidationFailureHandler : Consumer<CatalogValidationFailure>
 
 /** Union type for all validation failures. */
 sealed interface CatalogValidationFailure {
-    val streamName: String
-    val streamNamespace: String?
+    val streamID: StreamIdentifier
```

Comment:
I like this a whole lot better than the previous 2-strings!

Reply:
yes it felt cathartic to delete these lines